config/rbac/role.yaml

```diff
@@ -111,6 +111,17 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Note that these permissions were missing and aren't related to adding hosted mode support.